src/components/dockerRegistry/Docker.tsx

```diff
@@ -182,57 +187,74 @@ function DockerForm({
     const [Isdefault, toggleDefault] = useState(isDefault);
     const [toggleCollapsedAdvancedRegistry, setToggleCollapsedAdvancedRegistry] = useState(false);
     const [certError, setCertInputError] = useState('');
-
-    let awsRegionMap = awsRegionList.reduce((agg, curr) => {
-        agg.set(curr.value, curr.name);
-        return agg;
-    }, new Map());
-
+    const registryTypeList = Object.keys(REGISTRY_TYPE_MAP).map((key) => REGISTRY_TYPE_MAP[key]);
```

Comment:
If we're trying to get the list of values then can we try ```Object.values``` instead of doing this.